api/docker/properties-builder.sh

```diff
@@ -33,7 +33,12 @@ dbusername=${SPRING_DATA_MONGODB_USERNAME:-dashboarduser}
 #Database Password - default is blank
 dbpassword=${SPRING_DATA_MONGODB_PASSWORD:-dbpassword}
 
-#API encryption key. Optional. See http://capitalone.github.io/Hygieia/setup.html#encryption-for-private-repos	
+#dbhostport 
```

Comment:
please remove this duplicate property which is commented out, It will cause confusion.